src/main/java/com/artipie/asto/fs/RxFile.java

```diff
@@ -57,14 +56,6 @@
      */
     private final FileSystem fls;
 
-    /**
-     * Ctor.
-     * @param file The wrapped file.
-     */
-    public RxFile(final Path file) {
```

Comment:
@Sammers21 maybe we can use some default Vertx FileSystem for default constructor to avoid writing same code in all adapters? E.g.:
```java
class RxFile {
  private static final FileSystem FS_DEFAULT = Vertx.fileSystem();

  public RxFile(final Path file) {
    this(file, FS_DEFAULT);
  }
}
```

Reply:
@g4s8 , the code you suggest won't compile.  The other thing is that we need to ensure that a vert.x instance is created per JVM. i.e. we need to initialize it only once.